features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/utils/Emoji.kt

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2024 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.element.android.features.messages.impl.utils
+
+import androidx.emoji2.text.EmojiCompat
+import timber.log.Timber
+
+fun String.containsOnlyEmojis(maxEmojis: Int = Integer.MAX_VALUE): Boolean {
```

Comment:
Can we add some docs?

Reply:
Had to remind myself how to write ~JavaDoc~ KDoc(?), but done